test/automation/src/sql/notebook.ts

```diff
@@ -22,6 +22,9 @@ export class Notebook {
 	async openFile(fileName: string): Promise<void> {
 		await this.quickAccess.openQuickAccess(fileName);
 		await this.quickInput.waitForQuickInputElements(names => names[0] === fileName);
+		this.code.isActiveElement('.quick-input-widget .quick-input-box input').catch(e => {
+			this.code.logger.log(e);
```

Comment:
The error message thrown by driver.isActiveElement() contains the current active element (https://github.com/microsoft/azuredatastudio/blob/6ff1e3866bcad2a8a639fdaf6e6fe4aeab49b2b2/src%2Fvs%2Fplatform%2Fdriver%2Fbrowser%2FbaseDriver.ts#L83)